docs/tutorials/everything-is-an-event.md

```diff
@@ -0,0 +1,125 @@
+``` 
+
+Part of our goal with the new experience is to allow users to have the rules provide by tracee, but also be able to 
+collect events from the kernel they might be interested. Let's change the default events tracee was installed with,
+by editing the tracee damemonset running on the cluster.
+
+``` 
+kubectl edit ds/tracee -n tracee-system 
+
+``` 
+
+Now we can go to the line below, which shows all events (rules) tracee has loaded, and add to it the execve syscall events.
+
+From:
+
+``` 
+- event=anti_debugging,aslr_inspection,cgroup_notify_on_release,cgroup_release_agent,core_pattern_modification,default_loader_mod,disk_mount,docker_abuse,dynamic_code_loading,fileless_execution,hidden_file_created,illegitimate_shell,k8s_api_connection,k8s_cert_theft,kernel_module_loading,ld_peload,process_vm_write_inject,proc_fops_hooking,proc_kcore_read,proc_mem_access,proc_mem_code_injection,ptrace_code_injection,rcd_modification,sched_debug_recon,scheduled_task_mod,stdio_over_socket,sudoers_modification,syscall_hooking,system_request_key_mod 
+
+``` 
+
+To (we added `execve` to the end of the list):
+ 
+
+``` 
+- event=anti_debugging,aslr_inspection,cgroup_notify_on_release,cgroup_release_agent,core_pattern_modification,default_loader_mod,disk_mount,docker_abuse,dynamic_code_loading,fileless_execution,hidden_file_created,illegitimate_shell,k8s_api_connection,k8s_cert_theft,kernel_module_loading,ld_peload,process_vm_write_inject,proc_fops_hooking,proc_kcore_read,proc_mem_access,proc_mem_code_injection,ptrace_code_injection,rcd_modification,sched_debug_recon,scheduled_task_mod,stdio_over_socket,sudoers_modification,syscall_hooking,system_request_key_mod,execve
+
+``` 
+
+Let's restart tracee, so it starts with the new configuration.
+
+``` 
+kubectl rollout restart ds/tracee -n tracee-system 
+
+``` 
+
+Now if you check the logs you will see the `execve` events that are happening on your cluster.
+
+``` 
+kubectl  logs -f ds/tracee -n tracee-system | grep execve 
+ 
+
+{"timestamp":1671035008366697931,"threadStartTime":1776459241935,"processorId":5,"processId":26734,"cgroupId":20433,"threadId":26734,"parentProcessId":21685,"hostProcessId":26734,"hostThreadId":26734,"hostParentProcessId":21685,"userId":1000,"mountNamespace":4026531841,"pidNamespace":4026531836,"processName":"bash","hostName":"josedonizetti-x","containerId":"","containerImage":"","containerName":"","podName":"","podNamespace":"","podUID":"","eventId":"59","eventName":"execve","argsNum":2,"returnValue":0,"stackAddresses":null,"contextFlags":{"containerStarted":false},"args":[{"name":"pathname","type":"const char*","value":"/usr/bin/grep"},{"name":"argv","type":"const char*const*","value":["grep","--color=auto","execve"]}]} 
+
+``` 
+
+Tracee supports lots of events, you can see a list by running:
+
+```
+kubectl  exec ds/tracee -n tracee-system -- /tracee/tracee --list
+```
+
+Try to experiment with them and let us know what you think on the github discussion or slack: LINK
```

Comment:
LINK should be updated with a proper link

Reply:
Yup, it is a placeholder, discussion was not created yet